### ISSUE TITLE
@hoangtnm Only the metric model's (model finetuned on kitti dataset) output is metric, meaning in meters. The other models output relative disparity (inverse depth) upto an unknown scale and unknown shift which needs to be computed using GT for example

### ISSUE THREAD

**Issue by ysz-leo** [LiheYoung/Depth-Anything#233]
@hoangtnm Only the metric model's (model finetuned on kitti dataset) output is metric, meaning in meters. The other models output relative disparity (inverse depth) upto an unknown scale and unknown shift which needs to be computed using GT for example

_Originally posted by @kishore-greddy in https://github.com/LiheYoung/Depth-Anything/issues/72#issuecomment-1930222283_

**Comment by ysz-leo** [comment]
What is the difference between the true depth calculated by relative depth and the true depth output by fine-tuning metric model? Is the result of fine-tuning the model more accurate than the calculated result?
